test/smoke.bats

```diff
@@ -120,3 +120,12 @@ load 'helper'
 
   assert_json_eq '.events | length' 6
 }
+
+@test "expected number of http client events captured" {
+
+  javac test/HttpClientTest.java
```

Comment:
The Java 13 `java` command can take a source file to compile and run. Can you take advantage of that to get rid of this invocation of `javac`?

Reply:
Since we are running smoke tests with java 8 , `javac` must be used